libdnf/transaction/Item.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2017-2018 Red Hat, Inc.
+ *
+ * Licensed under the GNU Lesser General Public License Version 2.1
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#ifndef LIBDNF_TRANSACTION_ITEM_HPP
+#define LIBDNF_TRANSACTION_ITEM_HPP
+
+#include <memory>
+#include <string>
+
+#include "../utils/sqlite3/Sqlite3.hpp"
+
+class Item;
+typedef std::shared_ptr< Item > ItemPtr;
+
+#include "Types.hpp"
+
+class Item {
```

Comment:
In global namespace? Why is not in libdnf namespace? I found more occurrence of this.